bundle/Resources/config/parameters.yml

```diff
@@ -0,0 +1,2 @@
+parameters:
+    netgen.enhanced_selection.form.handler.expanded: false
```

Comment:
The format of the parameter is not correct for config resolver. It needs to be `namespace`.`scope`.`param_name`. Also, I would use `netgen_enhanced_selection` as namespace and `form_handler.expanded` for param name to keep it simpler a bit, so the parameter should be named `netgen_enhanced_selection.default.form_handler.expanded`.

Missing EOF too, btw :)